src/redux/books/books.js

```diff
@@ -1,27 +1,19 @@
-const ADD_BOOK = 'REACT-BOOKSTORE/src/redux/books/ADD_BOOK';
-const REMOVE_BOOK = 'REACT-BOOKSTORE/src/redux/books/REMOVE_BOOK';
+/* eslint-disable */
```

Comment:
- [ ] Kindly remove this line to enable linters and try to fix them to make your code better.